critiquebrainz/frontend/static/scripts/rating.js

```diff
@@ -0,0 +1,157 @@
+(function ($) {
+  'use strict';
+
+  var clearClass = 'rating-clear',
```

Comment:
I think it's just [`let`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let) by default now unless `var` is actually needed.